dawg_python/dawgs.py

```diff
@@ -315,9 +346,52 @@ def iteritems(self, prefix=""):
 
         while completer.next():
             key, value = completer.key.split(self._payload_separator)
-            item = (key.decode('utf8'), a2b_base64(bytes(value))) # bytes() cast is a python 2.6 fix
+            # bytes() cast is a python 2.6 fix
+            item = (key.decode('utf8'), a2b_base64(bytes(value)))
             yield item
 
+    def edges(self, prefix=""):
```

Comment:
It's similar data for all. It never returns a list of strings. It always returns a list of 2-tuples. For dawgs with no data, the tuples are `(str, True)` for terminal edges and `(str, False)` for non-terminals.

For dawgs with data, they're `(str, data)` for terminal edges, and `(str, False)` for non-terminals. Since data evaluates to true in a boolean situation, this seems most logical to me. If you want the data in an edge, you have it. If you want to just use the edges and know whether they're terminals or not, you can do that the same way across dawgs.